examples/combined_ODNP.py

```diff
@@ -116,9 +105,8 @@
     DNP_thermal_done = time.time()
     time_axis_coords = DNP_data.getaxis("indirect")
     time_axis_coords[0]["start_times"] = DNP_ini_time
-    DNP_data.set_prop("Ep_start_time", DNP_ini_time)
-    DNP_data.set_prop("Ep_thermal_done_time", DNP_thermal_done)
     time_axis_coords[0]["stop_times"] = DNP_thermal_done
+    DNP_data = DNP_data['nScans',-1:]
```

Comment:
toss everything but the last scan of the thermal which should be by this time "settled"